sotodlib/preprocess/processes.py

```diff
@@ -860,16 +860,22 @@ class GlitchFill(_Preprocess):
 
     def __init__(self, step_cfgs):
         self.signal = step_cfgs.get('signal', 'signal')
-        self.flag_aman = step_cfgs.get('flag_aman', 'glitches')
-        self.flag = step_cfgs.get('flag', 'glitch_flags')
+        self.flag_aman = step_cfgs.get('flag_aman')
+        self.flag = step_cfgs.get('flag')
 
         super().__init__(step_cfgs)
 
     def process(self, aman, proc_aman):
-        tod_ops.gapfill.fill_glitches(
-            aman, signal=aman[self.signal],
-            glitch_flags=proc_aman[self.flag_aman][self.flag],
-            **self.process_cfgs)
+        if (self.flag_aman is not None) and (self.flag is not None):
+            glitch_flags=proc_aman[self.flag_aman][self.flag]
```

Comment:
I think this `flag_aman` and `flag` could be replaced with a single argument if we use `attrgetter`

Reply:
Updated. This will force to change old configs.